.cookiecutter/includes/tox/passenv

```diff
@@ -0,0 +1,2 @@
+HUBSPOT_API_KEY
+TEST_DATABASE_URL
```

Comment:
These are the custom envvars that are [currently in `tox.ini`'s `passenv`](https://github.com/hypothesis/report/blob/f309dde299884d2e6370f96fda896c6ea7f5d069/tox.ini#L30-L31). The cookiecutter requires you to put them in this include file.

Probably the cookiecutter should put `TEST_DATABASE_URL` in `passenv` by default?